tests/integration/search/test_search_views.py

```diff
@@ -466,3 +466,38 @@ def test_search_hep_author_publication_with_not_existing_facet_name(isolated_api
         assert aggregation in result_aggregations
 
     assert len(expected_aggregations) == len(result_aggregations)
+
```

Comment:
the test ``test_search_facets_logs_with_query`` https://github.com/inspirehep/inspire-next/pull/3698/files#diff-498984cd0b86de92629fee84a5b9e6c8R255 should fail after this changes (I cannot comment on the line)

Reply:
Those tests are unreliable... :cry:

Reply:
it doesn't fail because the query doesn't use any of the facets I changed